packages/fuels/tests/scripts/script_raw_slice/src/main.sw

```diff
@@ -0,0 +1,16 @@
+script;
+
+fn main(length: u64) -> raw_slice {
+    let mut vec = Vec::new();
+    vec.push(42);
```

Comment:
I'm not sure if we gain much by randomizing the elements here, except in complexity.

Perhaps a single monotonically-increasing vector with the last element being `length`-1:

1 -> 0
2 -> 0,1
...
n -> 0, 1, ..., n-1

It's not all zeroes so we're covered on that front, and I doubt something will create monotonically increasing junk data.